README.md

```diff
@@ -22,7 +22,7 @@ For general PowerVS usage instructions please refer to the following links:
 - https://www.youtube.com/playlist?list=PLVrJaTKVPbKM_9HU8fm4QsklgzLGUwFpv
 
 
-:heavy_exclamation_mark: **Ensure that you're working from the desired release-specific version of this `README` as you deploy by first selecting the appropriate branch from the drop-down above. At the time of this writing, the latest release is 4.5: [README](https://github.com/ocp-power-automation/ocp4-upi-powervs/tree/release-4.5). The `README` document located in the `master` branch changes frequently and should not be considered final.**
+:information_source: **This branch must be used with latest OCP pre-release versions only. For stable releases please checkout specific release branches - {release-4.5, release-4.6 ...} and follow the docs.**
```

Comment:
Can we say `...and follow the respective README.` Better to give the README links here?

Reply:
It'll be hard to provide links to every release branch. We'll add 4.7 as well. IMHO let it be for now and we can revisit later